src/odig.ml

```diff
@@ -0,0 +1,59 @@
+open Import
+
+type t = Opam.t * Opam.Switch.t
+
+type of_opam_error = Odig_not_installed
+
+let of_opam (opam, switch) =
+  let open Promise.Syntax in
+  let+ is_installed = Opam.has_package opam switch "odig" in
```

Comment:
Why do we care if the `odig` package is installed? Isn't it sufficient for an `odig` binary to be present?

Reply:
Definitely makes sense indeed. I'll make the changes, thanks 😊

Reply:
https://github.com/ocamllabs/vscode-ocaml-platform/pull/771/commits/457797fef7b602debc00c3948fae777d6ad01b93